pkg/vf/virtio.go

```diff
@@ -275,7 +275,9 @@ func (config *StorageConfig) toVz() (vz.StorageDeviceAttachment, error) {
 	if config.ImagePath == "" {
 		return nil, fmt.Errorf("missing mandatory 'path' option for %s device", config.DevName)
 	}
-	return vz.NewDiskImageStorageDeviceAttachment(config.ImagePath, config.ReadOnly)
+	syncMode := vz.DiskImageSynchronizationModeFsync
```

Comment:
In the future it'd make sense I guess to plumb through these options as API too, but given the current foot-gun nature of things, let's hardcode for now.